localstack/services/cloudformation/models/ec2.py

```diff
@@ -530,6 +531,13 @@ def add_defaults(resource, stack_name: str):
     @staticmethod
     def get_deploy_templates():
         # TODO: validate again
+
+        def get_user_data_decoded(
+            properties: dict, logical_resource_id: str, resource: dict, stack_name: str
```

Comment:
With #8822 merged, all callbacks get two additional args passed:

```suggestion
            account_id: str, region_name: str, properties: dict, logical_resource_id: str, resource: dict, stack_name: str
```